overwatch/src/utils/traits.rs

```diff
@@ -0,0 +1,3 @@
+pub trait AsRef<T> {
+    fn as_ref() -> &'static T;
```

Comment:
Didn't we have a `ServiceId` type alias? :thinking:

Reply:
You mean instead of `T`?

Reply:
Actually, the comment was a brain-fart.